packages/paste-website/src/pages/components/table/index.mdx

```diff
@@ -287,58 +343,662 @@ Table rows (`Tr`) can be set with either top, middle, or bottom vertical alignme
+          <Text as="p" marginLeft="space30">Invalid</Text>
+        </Box>
+        <Text as="p" color="colorTextWeak" marginLeft="space60">2020-09-17, 16:24:28 PDT</Text>
+      </Td>
+      <Td>
+        <Text as="span" fontFamily="fontFamilyCode">
+          SMxarke3v30fv17hauqn86a7nhgm3b5d87
+        </Text>
+      </Td>
+      <Td>kevin@hotmail.com</Td>
+    </Tr>
+  </TBody>
+</Table>`}
+</LivePreview>
+
+### Table with tooltips on headers
+
+A `Th` can be composed to include [icon buttons](/components/button#link-style-button), such as an information icon button with additional
+details placed in a [Tooltip](/components/tooltip/) or [Popover](/components/popover/).
+
+<LivePreview scope={{Table, THead, TBody, Tr, Th, Td, Text, Box, Button, Tooltip, InformationIcon}} language="jsx">
+  {`<Table>
+  <THead>
+    <Tr>
+      <Th>Date</Th>
+      <Th>
+        <Box display="flex" alignItems="center">
+          <Text as="span" marginRight="space10">SID</Text>
+          <Tooltip text="SID Number">
+            <Button variant="reset" size="reset">
+              <InformationIcon decorative={false} title="Open Tooltip" display="block" />
+            </Button>
+          </Tooltip>
+        </Box>
+      </Th>
+      <Th>From</Th>
+    </Tr>
+  </THead>
+  <TBody>
+    <Tr>
+      <Td>
+        <Text as="p">16:24:28 PDT</Text>
+        <Text as="p">2020-09-17</Text>
+      </Td>
+      <Td>
+        <Text as="span" fontFamily="fontFamilyCode">
+          SM0yc4mxi6cn4z13bte7qmflc2drc85mlp
+        </Text>
+      </Td>
+      <Td>(602) 609-6747</Td>
+    </Tr>
+    <Tr>
+      <Td>
+        <Text as="p">16:24:28 PDT</Text>
+        <Text as="p">2020-09-17</Text>
+      </Td>
+      <Td>
+        <Text as="span" fontFamily="fontFamilyCode">
+          SMl29llgoihx286uhxfb0yc5n0sg391x5n
+        </Text>
+      </Td>
+      <Td>(602) 609-6747</Td>
+    </Tr>
+    <Tr>
+      <Td>
+        <Text as="p">16:24:28 PDT</Text>
+        <Text as="p">2020-09-17</Text>
+      </Td>
+      <Td>
+        <Text as="span" fontFamily="fontFamilyCode">
+          SMxarke3v30fv17hauqn86a7nhgm3b5d87
+        </Text>
+      </Td>
+      <Td>(602) 609-6747</Td>
+    </Tr>
+  </TBody>
+</Table>`}
+</LivePreview>
+
+### Table with truncated cells
+
+In rare cases, you may want to [truncate](/components/truncate) the text in a given cell. For example, when a table contains
+long URLs, and the user does not need to read the full URL.
+
+<LivePreview scope={{Table, THead, TBody, TFoot, Tr, Th, Td, Anchor, Truncate}} language="jsx">
+  {`<Table tableLayout="fixed">
+  <THead>
+    <Tr>
+      <Th>Agents</Th>
+      <Th>Skills</Th>
+      <Th>Profile URL</Th>
+    </Tr>
+  </THead>
+  <TBody>
+    <Tr>
+      <Th scope="row">Adam Brown</Th>
+      <Td>English, French, Sales, Spanish</Td>
+      <Td>
+        <Anchor href="/getting-started/about-paste">
+          <Truncate>https://paste.twilio.design/getting-started/about-paste</Truncate>
```

Comment:
@richbachman do you know why these links aren't underlined?

Reply:
Just had the order of the components wrong. Good catch. Will be fixed in my next push: https://share.getcloudapp.com/BluQlg1g